integrations/nifi/nifi-nar-bundles/nifi-standard-services/nifi-hadoop-service-bundle/nifi-hadoop-service/src/main/java/com/thinkbiganalytics/nifi/v2/thrift/HivePoolableConnectionFactory.java

```diff
@@ -0,0 +1,150 @@
+package com.thinkbiganalytics.nifi.v2.thrift;
+
+/*-
+ * #%L
+ * kylo-nifi-hadoop-service
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)